timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -226,7 +227,7 @@ def _get_estimator_init_args(self) -> Dict[str, Any]:
         `self._get_model_params` for better readability."""
         return self._get_model_params()
 
-    def _get_estimator(self) -> GluonTSEstimator:
```

Comment:
Isn't this too big a change for the task we're solving? We already have a method for getting init_args for estimators, why should we need to pass other args into this method, and especially for all models?